policyengine-client/src/common/accounting.jsx

```diff
@@ -0,0 +1,31 @@
+import { Table } from "antd";
+
+export default function AccountingTable(props) {
+    const columns = [
+        { title: "Component", dataIndex: "label", width: 300 },
+        { title: "Baseline", dataIndex: "baseline" },
+        { title: "Reform", dataIndex: "reformed" },
+    ];
+    console.log(props)
+    const filteredVariables = props.variableNames.map(name => props.variables[name]).filter(variable => Object.keys(variable).includes("accounting"));
+    let output = filteredVariables.map(variable => Object.assign(variable, {
+        key: variable.title,
+        label: (!variable.accounting.is_addition ? " - " : "") + variable.title,
```

Comment:
Could we make the `is_addition` negative logic to the values themselves?

Reply:
Not sure I follow - we do need to specify that income tax is subtracted rather than added

Reply:
Yes I'm suggesting that this shows up as `-2,602`

Reply:
Oh right, yep got it